editor/js/reaction.js

```diff
@@ -239,6 +286,11 @@ function downloadReaction() {
   xhr.send(binary);
 }
 
+/**
+ * Downloads a dataset as a serialized Dataset proto.
+ * @param {string} fileName The name of the dataset to fetch.
+ * @return {!Uint8Array}
```

Comment:
It's more correct to say that this function returns a `Promise`. It seems like there is a lot of discussion around how to document type when it is fulfilled or rejected, but no clear consensus. Google's guide seems to suggest you would want to use `!Promise<!Uint8Array>` here

Reply:
Done.